test/test-ghe-restore-parallel.sh

```diff
@@ -4,6 +4,12 @@ set -e
 
 export GHE_PARALLEL_ENABLED=yes
 
+# use temp dir to fix rsync file issues in parallel execution:
+# we are imitating remote server by local files, and running rsync in parallel may cause
+# race conditions when two processes writing to same folder
+tempdir=$(mktemp -d -t backup-utils-restore-temp-XXXXXX)
```

Comment:
Use a different variable name here for good measure. tempdir is common across the code and we do a lot of sourcing unfortunately.

Reply:
Changed this to `parallel_rsync_tempdir`.